DQMOffline/Trigger/interface/HLTTauDQMOfflineSource.h

```diff
@@ -55,6 +58,10 @@ class HLTTauDQMOfflineSource : public DQMEDAnalyzer {
       edm::EDGetTokenT<LVColl> token;
     };
     std::vector<RefObject> refObjects_;
+    std::vector<edm::ParameterSet> tagAndProbePaths;
```

Comment:
Hi @slehti - it doesn't look like this object needs to be member data of the class - it could instead be a local variable?